libs/@local/hql/cst/src/expr/mod.rs

```diff
@@ -0,0 +1,29 @@
+pub mod call;
+pub mod constant;
+pub mod path;
+pub mod signature;
+
+use hql_span::SpanId;
+
+use self::{call::Call, constant::Constant, path::Path, signature::Signature};
+use crate::Spanned;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ExprKind<'arena, 'source> {
+    Call(Call<'arena, 'source>),
+    Signature(Signature<'arena>),
+    Path(Path<'arena>),
+    Constant(Constant<'arena, 'source>),
+}
```

Comment:
I find the name misleading, I'd expect `kind` be small (`Copy`), but it's just the inner expression without the span ID. I'm not able to come up with a better name, though, so I won't block the PR on this but maybe you have an idea.

Reply:
Yea me too, but I couldn't find a better name for it, so I stuck with that one for now 😅 .